### ISSUE TITLE
Unable to read Tables in Dark Mode

### ISSUE THREAD

**Issue by clapierre** [w3c-cg/publ-a11y#813]
### Describe the problem

When selecting Dark Mode the banded tables can not be read, insufficient color contrast.

<img width="1756" height="945" alt="screenshot: Accessibility Metadata Features Table shown in dark mode, light blue background with light gray text." src="https://github.com/user-attachments/assets/62b77541-f3d5-4fde-beb2-3d3ccbfd9f9f" />

**Comment by mattgarrish** [comment]
The epub spec use the same "zebra" styling for tables but the font colour stays black in dark mode. That gives is an 18.5:1 ratio even if the light blue background in dark mode looks a bit funky. Try reading this from the RS spec:

<img width="1152" height="879" alt="Image" src="https://github.com/user-attachments/assets/bd644318-c8a9-4c03-aeba-b2f36f02922b" />

There must be a fix for the colour in the CSS that we can steal and use here, but maybe we should change the alternating background while we're at it, and restore some kind of header row colour? @iherman?

**Comment by iherman** [comment]
@mattgarrish the dark mode version of the 'core' style shit is a bit raw here and there, that is why I have added some stuff in our common.css file. I am only happy if that is improved.

But... I know I have already seen the output that @clapierre shows, and I am not sure why and when it happens. I *think* that, for example, it goes wrong when a particular area is selected, but I am not sure. @clapierre is it always so, systematically, or only sometimes?

**Comment by clapierre** [comment]
@iherman it seems to only appear when there is the Light/Dark/Auto option in the left sidebar.

Fore example when I go to the same spec in Chrome where i have dark mode enabled. it seems to work fine.

See the image below

<img width="1626" height="721" alt="screenshot: Chrome darkmode setting table is fully readable." src="https://github.com/user-attachments/assets/caea5992-3725-4cd5-bff6-f8b3c76c0049" />

here is a screenshot where this Light/Dark/Auto mode is included where the TOC appears etc.

<img width="943" height="807" alt="screenshot: with the Light/Dark/Auto present in sidebar" src="https://github.com/user-attachments/assets/3b92b1eb-1d38-4907-be33-57b3fa650adb" />

**Comment by mattgarrish** [comment]
I get the grey text pretty consistently in the crosswalk and the black in the core specs. Going by the computed styles, it's coming from the W3C base.css file, specifically a body rule that sets a variable for the text colour:

color: var(--text);

(at line 188).

Looking at common.css in the epub spec repository we have this rule that overrides it:

body.darkmode table.zebra tr:nth-child(even) {
    color: black;
}

**Comment by mattgarrish** [comment]
> the dark mode version of the 'core' style shit is a bit raw here

LOL, apt if unintended... 😉
